### PR TITLE
Various PF fixes

### PR DESCRIPTION
**Pull request by jelly**
Deprecations / missing title found by making extranous `console.error` fail. Split up from https://github.com/cockpit-project/cockpit/pull/19053

### CODE REVIEW

File: pkg/systemd/reporting.jsx [hunk 1]
```diff
@@ -226,6 +226,7 @@ class BusWorkflowRow extends React.Component {
                         this.setState({ message: _("Waiting for input…") });
                         const task_prompt = client.proxy("org.freedesktop.reportd.Task.Prompt", object_path);
                         const props = {
+                            title: _("Report to Fedora?"),
```
Comment (martinpitt) [inline pkg/systemd/reporting.jsx:229]: Is this justified? Shouldn't "Fedora" come from some metadata of the backend?

**Review by martinpitt — COMMENTED** [review]
This looks good by and large, but I'd like to at least discuss the hardcoded "Fedora" string. Maybe it's okay :-)

File: pkg/systemd/reporting.jsx (hunk 1 above)
Comment (jelly) [reply pkg/systemd/reporting.jsx:229]: Oh good point! I think the text comes from reportd/taskd so this is :100: wrong.

**Comment by jelly** [comment]
> This looks good by and large, but I'd like to at least discuss the hardcoded "Fedora" string. Maybe it's okay :-)

image

So this is where my confusion started :)

**Comment by jelly** [comment]
> > This looks good by and large, but I'd like to at least discuss the hardcoded "Fedora" string. Maybe it's okay :-)
> 
> image
> 
> So this is where my confusion started :)

`Report to Fedora`  comes from some label.

> > This looks good by and large, but I'd like to at least discuss the hardcoded "Fedora" string. Maybe it's okay :-)
> 
> image
> 
> So this is where my confusion started :)

More meh, we should maybe just fix the UX here. You can't report without an API key. So maybe `Report` should be disabled and we should show a setup button first? @garrett (We have an open issue for reportd)

https://github.com/cockpit-project/cockpit/issues/19059

So I'll drop this commit here, let's focus on making something better and adding a `Title`.

image

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**